test/integration/firewalls_test.go

```diff
@@ -19,7 +20,7 @@ var (
 func TestListFirewalls(t *testing.T) {
 	client, _, teardown, err := setupFirewall(t, []firewallModifier{
 		func(createOpts *linodego.FirewallCreateOptions) {
-			createOpts.Label = randString(12, lowerBytes, digits) + "-linodego-testing"
+			createOpts.Label = randString(12, lowerBytes, upperBytes) + "-linodego-testing"
```

Comment:
A Firewall's label cannot begin with a number, rerunning this threw an error because of it.